src/Meziantou.Analyzer/Rules/PreferReturningCollectionAbstractionInsteadOfImplementationAnalyzer.cs

```diff
@@ -49,6 +49,7 @@ public AnalyzerContext(Compilation compilation)
 
             XmlClassAttributeSymbols.AddIfNotNull(compilation.GetBestTypeByMetadataName("System.Xml.Serialization.XmlTypeAttribute"));
             XmlClassAttributeSymbols.AddIfNotNull(compilation.GetBestTypeByMetadataName("System.Xml.Serialization.XmlRootAttribute"));
+            XmlClassAttributeSymbols.AddIfNotNull(compilation.GetBestTypeByMetadataName("System.SerializableAttribute"));
```

Comment:
`SerializableAttribute` is for the Binary serializer, not for xml serializer. You can create a new property similar to `XmlClassAttributeSymbols`.

```
BinarySerializerClassAttributeSymbols.AddIfNotNull(compilation.GetBestTypeByMetadataName("System.SerializableAttribute"));
```